test/AST-Quake/custom_op_concrete_matrix.cpp

```diff
@@ -6,7 +6,7 @@
  * the terms of the Apache License 2.0 which accompanies this distribution.    *
  ******************************************************************************/
 
-// RUN: cudaq-quake %cpp_std %s | cudaq-opt -lift-array-value -get-concrete-matrix | FileCheck %s
+// RUN: cudaq-quake %cpp_std %s | cudaq-opt -const-prop-complex -lift-array-value -get-concrete-matrix | FileCheck %s
```

Comment:
This doesn't appear to change the test? Is it not needed?

Reply:
The test failed without it because it needs some complex const prop opts that I moved from `lift-array-value` to `const-prop-complex`